portal-web/docroot/html/taglib/ui/search_iterator/deprecated/list.jsp

```diff
@@ -145,12 +148,22 @@ if (iteratorURL != null) {
 						<c:if test="<%= (rowChecker != null) && (i == 0) %>">
 							width="1%"
 						</c:if>
+
+						<c:if test="<%= (orderKey != null) && orderCurrentHeader %>">
+							aria-live="assertive" aria-sort="<%= LanguageUtil.get(resourceBundle, sortedActionMessageKey) %>"
```

Comment:
@marcoscv-work @drakonux  There's not many places in the product with `aria-live="assertive"`. I've only seen them when reporting errors. 
Do we have any recommendations or guidance about using `assertive` vs `polite`?